package.json

```diff
@@ -15,15 +20,15 @@
   "collection:main": "dist/collection/index.js",
   "unpkg": "dist/mui-core/mui-core.js",
   "files": [
-    "dist/",
-    "loader/"
+    "dist/"
   ],
   "scripts": {
     "build": "npm run update-version && stencil build",
     "bundlesize": "npm run build && npx bundlesize",
     "start": "stencil build --dev --watch --serve",
     "generate": "stencil generate",
     "lint": "eslint --ext .js,.ts,.tsx src",
```

Comment:
See other comment about the `prepare` npm script. If you add that, you could also `npm version --from-git` like so:

```json
  "prepare": "npm run build && npm version --from-git"
```

Then I believe all that would happen before `npm publish`. Alternately, you can use `prepublishOnly` if we only need to run this before pushing to npm… maybe that would be better? Not sure